src/lib/Components/Consignments/Submission/__tests__/create-tests.ts

```diff
@@ -0,0 +1,29 @@
+jest.mock("../../../../metaphysics", () => ({ metaphysics: jest.fn() }))
+
+import { metaphysics } from "../../../../metaphysics"
+import { withPhotos } from "../../__stories__/consignmentSetups"
+
+import create from "../create"
+
+const mockphysics = metaphysics as jest.Mock<any>
+
+it("makes a graphQL request to metaphysics", async () => {
+  mockphysics.mockImplementationOnce(() =>
+    Promise.resolve({
+      data: {
+        createConsignmentSubmission: {
+          submission: {
+            id: "1234",
+            artist_id: "asdasd",
+          },
+        },
+      },
+    })
+  )
+
+  const response = await create(withPhotos)
+  const query = mockphysics.mock.calls[0][0].query
+
+  expect(query).toContain("mutation")
+  expect(query).toContain("createConsignmentSubmission")
+})
```

Comment:
The other option would be to compare against a snapshot, it's a long string that's a bit awkward, so instead I opted to test for the real things I care about